jafarabdi.tf

```diff
@@ -0,0 +1,18 @@
+locals {
+  jafarabdi_team = [
+    "JafarAbdi",
+  ]
+
+  jafarabdi_repositories = [
+    "feetech_ros2_driver-release",
+    "jacro-release",
```

Comment:
Will these repos automatically get created?

Reply:
Yes these will get created when the PR is merged and terraform is deployed. Usually the ros2-gbp maintainer will deploy as soon as they finish a batch of reviews and comment on each deployed issue/PR that the deployment has completed which means that added resources should be available.